CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.3.0] - 2019-04-01
```

Comment:
This should probably be 4.0.0, since it isn't backwards compatible in terms of signing formats or py35.